cypress/locators/drawer/locators.js

```diff
@@ -0,0 +1,5 @@
+// component preview locators
+export const DRAWER = '[data-element="drawer"]';
+export const DRAWER_TOGGLE = '[data-element="drawer-toggle"]';
+export const DRAWER_SIDEBAR = '[data-element="drawer"] div:nth-child(1)';
```

Comment:
move it to `index.js` file as
`export const drawerSidebar = () => cy.iFrame(DRAWER).find(div:nth-child(1));`

Reply:
fixed 4d965d0